website/content/api-docs/secret/databases/index.mdx

```diff
@@ -52,6 +52,10 @@ list of additional parameters.
   for this database. If not specified, this will use a default policy defined as:
   20 characters with at least 1 uppercase, 1 lowercase, 1 number, and 1 dash character.
 
+- `disable_escaping` `(boolean: false)` - Determines whether special characters in the
```

Comment:
I am wondering if this should go under "Common fields" below. I don't have a strong preference here though

Reply:
The reason I put it in the top list was because the description under common fields says "several common fields you will see across many database plugins... reference individual plugin documentation for details on whether the fields are supported".

Since `disable_escaping` should exist for all plugins, I thought the top matched better maybe. Although, putting it in the section where username and password are listed also make sense. So I'm really not sure.

I'm not quite sure why there are 2 lists like this to begin with rather than 1. Perhaps @calvn or @austingebauer might be able to explain this better?

Reply:
Since `disable_escaping` is something that's under `SQLConnectionProducer`, It might make sense to add it to the common fields sections, and add documentation on the databases that support this parameter. The DB's that do support this field are the one that uses `SQLConnectionProducer`. For instance, Cassandra implements and uses its own connection producer, so I don't think it would support this field.

Here's the list of DB's I found that uses `SQLConnectionProducer`. There may be other's so it'd be good to cross-check.

- hana
- mssql
- postgres
- redshift
- mysql
- oracle

Reply:
That sounds like a good idea, thanks for pointing out SQLConnectionProducer's role in this. Looking through the codebases I also see snowflake uses it. I'm adding this now to get this pushed and merged today.

Reply:
Here are the updated pages to make it easy to check

Overview common fields:
https://vault-4cepfxddf-hashicorp.vercel.app/api-docs/secret/databases#common-fields

Plugins:
* hana https://vault-4cepfxddf-hashicorp.vercel.app/api-docs/secret/databases/hanadb#disable_escaping
* mssql https://vault-4cepfxddf-hashicorp.vercel.app/api-docs/secret/databases/mssql#disable_escaping
* postgres https://vault-4cepfxddf-hashicorp.vercel.app/api-docs/secret/databases/postgresql#disable_escaping
* redshift https://vault-4cepfxddf-hashicorp.vercel.app/api-docs/secret/databases/redshift#disable_escaping
* mysql https://vault-4cepfxddf-hashicorp.vercel.app/api-docs/secret/databases/mysql-maria#disable_escaping
* oracle https://vault-4cepfxddf-hashicorp.vercel.app/api-docs/secret/databases/oracle#disable_escaping
* snowflake https://vault-4cepfxddf-hashicorp.vercel.app/api-docs/secret/databases/snowflake#disable_escaping